### ISSUE TITLE
any chance to remove dependency on 'blob'?

### ISSUE THREAD

**Issue by amitport** [socketio/engine.io-parser#99]
relevant to socket.io issue https://github.com/socketio/socket.io-client/issues/1166

published https://github.com/webmodules/blob/pull/14, but not getting any response (+last commit to `blob` was in 2015)

@darrachequesne @rauchg @nkzawa

**Comment by darrachequesne** [comment]
Hi @amitport, it seems you're now owner of the `blob` package :tada:
```
$ npm owner ls blob
amitport <amit.portnoy@gmail.com>
rase- <tonykovanen@hotmail.com>
```
Could you please publish a new release?

**Comment by amitport** [comment]
Published [blob@0.0.5](https://github.com/amitport/blob/releases/tag/v0.0.5)!

**Comment by jimmywarting** [comment]
Can we not simply remove the blob dependency and some other old legacy stuff?

I mean Blob and arrayBuffer is supported now in all "green" browser version... just a question, what browsers are you giving support to? MS stop giving support to IE < 11. Are you going to do the same?

Also do we still want to do PhantomJS relevant stuff? there is no page visitor that use phantomJS. only developers who want to do testing. so 99.9% of all users won't even care
```js
var isPhantomJS = typeof navigator !== 'undefined' && /PhantomJS/i.test(navigator.userAgent);
```
Now days most ppl use headless chrome
